packages/wmr/src/plugins/html-entries-plugin.js

```diff
@@ -97,7 +98,16 @@ export default function htmlEntriesPlugin({ root, publicPath }) {
 					all.push(ref);
 					waiting.push(
 						fs.readFile(abs, 'utf-8').then(source => {
-							this.setAssetSource(ref, source);
+							if (/\.s[ac]ss$/.test(abs)) {
+								transformSass.call(this, abs, abs, source, root, true, sourcemap).then(result => {
```

Comment:
Moved the `generateBundle` call from `sass-plugin` here as this ensures that we resolve imported files correctly during transformation. Else we emit the scss asset here as an asset and will be transforming in the dist-folder